<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoFormatterWidget</class>
 <widget class="QWidget" name="ProtoFormatterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="styleComboBox">
       <item>
        <property name="text">
         <string>LLVM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GNU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Google</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chromium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Microsoft</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mozilla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WebKit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="formatButton">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="toolTip">
        <string>Open</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCopy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CodeEditor" name="codeEditor"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
